### DISCUSSION TITLE
wis2box 0.4.0 released

### DISCUSSION THREAD

**Discussion by tomkralidis** [wmo-im/wis2box#276] [Announcements]
This is the 0.4.0 working release of wis2box. Updated capabilities include:

- numerous UI/UX updates in support of user workflow
- numerous updates to user documentation
- numerous updates to WIS2 topic hierarchy workflow
- initial implementation of streamlined download for user
- AsyncAPI support added to API
- numerous BUFR pipeline updates (WSI/TSI handling, workflow, publishing, updated BUFR tables)
- removal of WebDAV support (given default support for S3)
- configurable basemaps

All contributions are always welcome. Feedback is welcome via GitHub [issues](https://github.com/wmo-im/wis2box/issues) or [discussions](https://github.com/wmo-im/wis2box/discussions).

The wis2box team

https://docs.wis2box.wis.wmo.int

<hr /><em>This discussion was created from the release <a href='https://github.com/wmo-im/wis2box/releases/tag/0.4.0'>0.4.0</a>.</em>

**Comment by samuelchau** [comment]
Hi wis2box team,

I've been experimenting with the latest wis2box software. 

After adding discovery metadata(*yml) and station_list.csv provided from the source code, and having spent some time testing around with the OAPI, I'd like to reset the software to its initial state.
Is there way to clear all the metadata/data from the wis2box and reset it to its initial state(without any provisioned metadata)?

I have tried manually deleting all the metadata/data from my $WIS2BOX_HOST_DATADIR and then run "wis2box-ctl.py stop" and then "wis2box-ctl.py start" to restart. But when I re-visit the landing page at http://localhost:8999, it's still displaying the data/metadata(with the Explore, Observations, Discover Metadata buttson) from the topic mwi.malawi_wmo_demo.data.core.weather.surface-based-observations.synop that I added previously.   

I even tried manually removing all the docker images and then rebuilt them and restarted it.
But still the landing page is showing the malawi surface observation data set that I previously added.

I don't know why the metadata are still kept in the cache after rebuilding all the docker images.

Any advice would be appreciated.

**Reply by samuelchau** [reply]
I have found the answer to my question.
After stopping wis2box with "wis2box-ctl.py stop",  the docker volumes created were still there. 
Once I removed all the docker volumes with "docker volume rm", and re-started wis2box, and I got the expected "empty" landing page at http://localhost:8999.
